invenio_workflows/api.py

```diff
@@ -53,7 +53,7 @@ def worker(self):
         try:
             return import_string('invenio_workflows.workers.%s:%s' % (
                 cfg['CFG_BIBWORKFLOW_WORKER'], cfg['CFG_BIBWORKFLOW_WORKER']))
-        except:
+        except:  # noqa
```

Comment:
I am not sure honestly. I'll add `Exception`, I just did not want to touch more than necessary with the PR. (Although I started moving the imports here due to the sphinx warnings in Travis, but it's everywhere it seems.)